### PR TITLE
Fixes #648 :Added my geolocation and social links

### LINKED ISSUE #648: Add geolocation
### Expected results
Geolocation present
### Actual results
My geolocation not there
### Steps to reproduce the issue
Add it
## Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #648 
My geolocation is added in this PR, also a social link

### CODE REVIEW

**Review by abishekvashok — APPROVED** [review]

File: _data/students.yml [hunk 1]
```diff
@@ -240,5 +243,6 @@
 - name: Yash Kothari
   country: India
   github: Beta-King
+  twitter: Beta_King_500
```
Comment (abishekvashok) [inline _data/students.yml:246]: Why was this added though?

**Comment by saarthakchats** [comment]
He's my classmate and good friend, I asked him if I should add one social link for him, he said yes. So...

**Comment by saarthakchats** [comment]
@abishekvashok

**Review by abishekvashok — APPROVED** [review]

File: _data/students.yml (hunk 1 above)
Comment (vaibhavsingh97) [inline _data/students.yml:246]: Two twitter accounts. why?

Comment (abishekvashok) [reply _data/students.yml:246]: Quote @saarthakchats 
> He's my classmate and good friend, I asked him if I should add one social link for him, he said yes. So...

**Review by codedsun — APPROVED** [review]

**Comment by saarthakchats** [comment]
@vaibhavsingh97 That guy is my friend
